cubic-server/Server.cpp

```diff
@@ -145,3 +160,42 @@ void Server::_stop()
         delete command;
     LINFO("Server stopped");
 }
+
+void Server::_downloadFile(const std::string &url, const std::string &path)
+{
+    if (std::filesystem::exists(path)) {
+        LDEBUG("File " << path << " already exists. Skipping download");
```

Comment:
```suggestion
        LDEBUG("File " << path << " already exists. Skipping download");
        return;
```
and then remove the else 💢

Reply:
I don't think that's needed to be fair